.gitattributes

```diff
@@ -19,9 +19,9 @@
 /composer.json export-ignore
 /composer.lock export-ignore
 /CONTRIBUTING.md export-ignore
-/Gruntfile.js export-ignore
+/gulpfile.mjs export-ignore
```

Comment:
confirming, this should be a replacement, not an adendum?

Reply:
Correct. Using Gulp _instead of_ Grunt (which we weren't really using for anything anyway.